contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/advisors/SitePropertyAdvisor.java

```diff
@@ -43,5 +52,28 @@ public boolean siteCanUseReviewService(Site site) {
 		
 		return false;
 	}
+	
+	public boolean siteCanUseLTIReviewService(Site site) {
+		//maybe need to add this too -- if(!siteCanUseReviewService(site)) return false;
+		ResourceProperties properties = site.getProperties();
+		
+		String prop = (String) properties.get(siteLTIProperty);
```

Comment:
This should be `getProperty(String)` so that you don't need the cast and so you can't ever get a classcast exception.